mantle/cmd/kola/testiso.go

```diff
@@ -293,20 +362,21 @@ func newQemuBuilder(outdir string) (*platform.QemuBuilder, *conf.Conf, error) {
 
 func newQemuBuilderWithDisk(outdir string) (*platform.QemuBuilder, *conf.Conf, error) {
 	builder, config, err := newQemuBuilder(outdir)
+
 	if err != nil {
 		return nil, nil, err
 	}
 
 	sectorSize := 0
-	if kola.QEMUOptions.Native4k {
+	if enable4k {
 		sectorSize = 4096
 	}
 
 	disk := platform.Disk{
 		Size:       "12G", // Arbitrary
 		SectorSize: sectorSize,
 
-		MultiPathDisk: kola.QEMUOptions.MultiPathDisk,
+		MultiPathDisk: enableMultipath,
```

Comment:
This blank line was in the code before, but I'm not sure why. Maybe we delete it?